src/components/Navigation/NavigationBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './NavigationBar.scss'
+
+export default function NavigationBar() {
+  return (
+    <nav className={styles.navigation}>
+      <ul>
+        <li>
+          <Link to="/subjects/"><i className ="far fa-lightbulb"></i>   subjects </Link>
+        </li>
+        <li>
+          <Link to='/bills/'><i className ="far fa-file-alt"></i> bills</Link>
+        </li>
+        <li>
+          <Link to='/politicians/'><i className ="far fa-handshake"></i>  politicians</Link>
```

Comment:
this is mainly a code style change. lets get rid of the space between `className` and `=`

```suggestion
          <Link to='/politicians/'><i className="far fa-handshake"></i>  politicians</Link>
```